tools/lint.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Comment:
Ah thanks @gsemet for taking time to review! I've a lot to learn regarding execution environments, I'll start by considering this!

#### TODO
- [ ] Read up on pipenv and figure out differences with venv and virtualenv.
- [ ] Ensure a robust execution environment

Reply:
The shebang here is already correct. While individuals are free to use pipenv, on CI since we already have an env, `pip install` is all that should be used.